--- /home/ec2-user/guidance-for-generative-bi-v1.9.0/application/utils/prompts/generate_prompt.py	2025-02-21 09:11:33.000000000 +0000
+++ /home/ec2-user/generative-bi-using-rag/application/utils/prompts/generate_prompt.py	2025-04-09 16:54:53.965149958 +0000
@@ -1074,8 +1074,11 @@
 <rules>
 1. answer should as concise as possible
 2. if you don't know the answer to the question, just answer you don't know.
+3. if entity information is provided, use it to enhance your answer
 </rules>
 
+{entity_info}
+
 <context>
 Here is a list of acronyms and their full names plus some comments, which may help you understand the context of the question.
 [{'Acronym': 'NDDC', 'Full name': 'Direct Digital Commerce'},
